### DISCUSSION TITLE
Deploy Storybook in Next.js Repo to Vercel but error functions config from vercel.json

### DISCUSSION THREAD

**Discussion by dickyindra** [vercel/next.js#49212] [Help]
### Summary

I have Storybook in Next.js repo.
I want to deploy to vercel.

But I got an error:
```
Error: The pattern "src/pages/**/*" defined in `functions` doesn't match any Serverless Functions inside the `api` directory.
unused_function: The pattern "src/pages/**/*" defined in `functions` doesn't match any Serverless Functions inside the `api` directory.
```

Here is my vercel.json config:
```
{
    "rewrites": [...],
    "headers": [...],
    "functions": {
        "src/pages/**/*": {
            "memory": 256
        }
    }
}
```

How to solve it?

**Comment by hjoelh** [comment]
@dickyindra Did you solve it? Just ran into this too

It seems to be caused by having `Framework Preset: Anything other than Next` (in our case Storybook) while also having vercel.json > functions > "src/pages/**/*" (anything other than /api/)

**Comment by hjoelh** [comment]
Fix for now:

(Project settings > Git)
Ignored Build Step
Custom: rm vercel.json && exit 1
